chapter_linear-networks/linear-regression_vn.md

```diff
@@ -657,29 +657,29 @@ d2l.plot(x, [normal(x, mu, sigma) for mu, sigma in parameters], xlabel='z',
 As you can see, changing the mean corresponds to a shift along the *x axis*, and increasing the variance spreads the distribution out, lowering its peak.
 -->
 
-Ta có thể thấy rằng, thay đổi giá trị trung bình tương ứng với việc dịch chuyển phân phối dọc theo *trục x*, đồng thời tăng giá trị phương sai sẽ trải rộng phân phối và làm giảm giá trị đỉnh của nó.
+Ta có thể thấy rằng, thay đổi giá trị trung bình tương ứng với việc dịch chuyển phân phối dọc theo *trục x*, tăng giá trị phương sai sẽ trải rộng phân phối và làm giảm giá trị đỉnh của nó.
 
 <!--
 One way to motivate linear regression with the mean squared error loss function is to formally assume that observations arise from noisy observations, where the noise is normally distributed as follows
 -->
 
-Để thấy rõ hơn mối quan hệ giữa hồi quy tuyến tính và hàm mất mát trung bình bình phương sai số (MSE), ta có thể giả định rằng các quan sát bắt nguồn từ những quan sát nhiễu, giá trị nhiễu này tuân theo phân phối chuẩn như sau:
+Để thấy rõ hơn mối quan hệ giữa hồi quy tuyến tính và hàm mất mát trung bình bình phương sai số (MSE), ta có thể giả định rằng các quan sát bắt nguồn từ những quan sát nhiễu, giá trị nhiễu này tuân theo phân phối chuẩn như sau: 
 
 $$y = \mathbf{w}^\top \mathbf{x} + b + \epsilon \text{ where } \epsilon \sim \mathcal{N}(0, \sigma^2).$$
 
 <!--
 Thus, we can now write out the *likelihood* of seeing a particular $y$ for a given $\mathbf{x}$ via
```

Comment:
Chỗ này mình bối rối quá, có cao nhân nào giúp với. 😕 😖 
Theo mình hiểu chỗ này tác giả viết sai rồi. Vậy nên ở đây mình mới phải dịch `likelihood` là `khả năng` (`probability`) trong khi dòng 682 ở dứoi dịch đúng là `hợp lý`.
P(y|x)  = L(x|y) là
 * *khả năng* thu được y, cho trước x : probability of y given x 
  * tính *hợp lý* của x, với dữ liệu quan sát được y : likelihood of x, given observed data y.
Vậy nên câu này đúng ra phải là  
```suggestion
Thus, we can now write out the *likelihood* $\mathbf{x}$ being true, given seeing a particular $y$ 
```
Mình search thì thấy có tài liệu này họ lại viết giống trong sách này (https://web.stanford.edu/class/msande226/maximum_likelihood.pdf)
![Screenshot 2020-03-04 at 16 09 25](https://user-images.githubusercontent.com/12573521/75898825-87ec2f80-5e32-11ea-92f2-b18d7262ea98.png)

Reply:
Chờ thêm các bạn bàn luận thêm về chỗ này. Nếu không có ý kiến gì thêm thì mình cứ giữ nguyên như ý tác giả rồi sửa sau nếu cần.

Reply:
Tài liệu đó viết đúng, vì `Y` của họ là các quan sát. 

Theo mình thì @lkhphuc nói đúng ở chỗ likelihood là nói về khả năng một sự kiện xảy ra cho trước một phân phối. Mà tác giả viết như trên nên có thể hiểu rằng `y` trong bài này thực chất là sự kiện chứ không phải phân phối.

![image](https://user-images.githubusercontent.com/23548268/76157586-334ff980-613d-11ea-9c4b-57ea7a65b545.png)

Nếu đọc kỹ thì sẽ thấy tác giả giả định `y` là sự kiện được lấy mẫu từ phân phối `wx + b + epsilon` nên viết như tác giả là hợp lý

Reply:
Tài liệu đó viết đúng, vì Y của họ là các quan sát.

Theo mình thì hiểu thì likelihood là nói về sự hợp lý của một quan sát xảy ra cho trước một phân phối. Mà tác giả viết như trên nên có thể hiểu rằng y trong bài này thực chất là quan sát chứ không phải phân phối.

Nếu đọc kỹ thì sẽ thấy tác giả giả định y là quan sát được lấy mẫu từ phân phối wx + b + epsilon nên viết như tác giả là hợp lý

Reply:
Sau một hồi bối rối thì có vẻ như tác giả viết ngược thật

Reply:
Nhờ các bạn trẻ thống nhất lại chỗ này rồi chúng ta merge.

Reply:
@lkhphuc bạn sửa lại nhé. Mà câu dịch ở dưới có cần sửa luôn không?

Reply:
Chỗ này câu dịch xài `khả năng` là mình thấy đọc đúng rồi. Mọi người cứ merge nhé.
Chỗ này chắc mấy ổng viết ngược hoặc là xài kí hiệu implicit gì đấy mình chưa rõ, thường mình thấy likelihood là P(data|parameters), mà chỗ này likelihood P(Y|X) cả Y và X đều là data cả.